api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -1414,22 +1414,23 @@ public void retireOrderFrequency_shouldRetireGivenOrderFrequency() throws Except
 	@Test
 	@Verifies(value = "should unretire given order frequency", method = "unretireOrderFrequency(OrderFrequency)")
 	public void unretireOrderFrequency_shouldUnretireGivenOrderFrequency() throws Exception {
-		OrderFrequency orderFrequency = orderService.getOrderFrequency(3);
+		executeDataSet("org/openmrs/api/include/OrderServiceTest-otherOrderFrequencies.xml");
+		OrderFrequency orderFrequency = orderService.getOrderFrequency(103);
 		assertNotNull(orderFrequency);
 		assertTrue(orderFrequency.isRetired());
-		assertEquals("Some Retire Reason", orderFrequency.getRetireReason());
```

Comment:
I had a look at https://github.com/openmrs/openmrs-core/blob/9b977a18fcc759af9ad782110b18efe27175dd76/api/src/test/resources/org/openmrs/include/standardTestDataset.xml  where the original dataset was located. The reason order frequency 3 is giving an error is because its class_id is not defined as 19 (line 19) i.e the the concept of order frequency 3 from the original dataset id not of class frequency.   On the other hand the one i have replaced it with (from file:///home/jthacker/openmrs-core/api/target/test-classes/org/openmrs/api/include/OrderServiceTest-otherOrderFrequencies.xml )  order frequency 103, its concept has class_id =19